### DISCUSSION TITLE
A more powerful image upscaler

### DISCUSSION THREAD

**Discussion by gin063** [lllyasviel/Fooocus#2662] [Ideas]
I found a powerful opensource upscaler, reverse engineering of the Magnific AI: 
https://github.com/philz1337x/clarity-upscaler
I wonder if the Fooocus team will add a more powerful upscaler in Fooocus?

**Comment by barepixels** [comment]
1) he didn't reverse engineer Magnific AI
2) he used a similar method that was published about 9 months ago https://www.youtube.com/watch?v=qde9f_U6agU
3) there are no innovative codes on his Github, last I checked all he did was upload A1111, that is it, wasting my time

If you want a powerful upscale look into SUPIR

With the help of SUPIR and Photoshop I took this:
before:  https://mega.nz/file/g1Y3HSyQ#R-HMq5PEGdLxQ2Or33nap9w0GiCueD3l4cYbtwBumXA
to: https://mega.nz/file/hk5FCLQC#TVgpcu8xJh-t0V4x7UwuOnOPs-KrcsMEa4FpAIktaKE    57MB

**Reply by gin063** [reply]
Thanks for your comment. I just watched this video, it’s exactly what you said, but TBH, this video is very difficult to watch😂
It seems that running SUPIR locally requires a very high hardware configuration, I only have a RTX 3060 laptop😭

**Comment by barepixels** [comment]
Yeah upscaling does require strong video cards.  It kinda justify why Magnific AI cost so much.  They have to rent some powerful servers

I recommend you use Topaz Gigapixels

Combine SUPIR and Topaz I took a 1024x512 pano to 10,000x5000 pixels as you can see here https://kuula.co/post/5n4bl

**Reply by gin063** [reply]
OK, thank you.
